lis/surfacemodels/land/ac.7.2/ac72_f2t.F90

```diff
@@ -204,10 +202,9 @@ subroutine AC72_f2t(n)
 
      ! Calculate Magnitude of Wind Speed (m/s)
      wind_tmp = SQRT(uwind(tid)**2 + vwind(tid)**2)
-     elevdiff = AC72_struc(n)%refz_uv - 2.
-     if (elevdiff.gt.0.1) then ! replace with corrected value
-        wind_tmp = wind_tmp * (4.87/LOG(67.8*AC72_struc(n)%refz_uv-5.42))
-     endif
+
+     ! Correct to 2 m if provided at another height
+     wind_tmp = wind_tmp * (4.87/LOG(67.8*AC72_struc(n)%refz_uv-5.42))
```

Comment:
This factor is ~0.7 if refs_uz = 2 m, meaning that mind_tmp>0 and always increases the final windspeed. Is there a reference for this equation, or should there be an if-statement to limit the increase only if refs_uv > 2 m?